tests/articulation_tests.ts

```diff
@@ -24,6 +26,7 @@ const ArticulationTests = {
   Start(): void {
     QUnit.module('Articulation');
     const run = VexFlowTests.runTests;
+    run('Articulation - Grace Notes', graceNotes);
```

Comment:
It would make more sense to place this test now on gracenote test becuase it combines grace notes not only with articulations but also with annotations and accidentals.

I suggest that you revert this test file from master and I open my PR again.

Reply:
ok